src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -22,6 +22,7 @@ describe('sidebar.components.sidebar-content', function() {
   let fakeAnalytics;
   let fakeAnnotations;
   let fakeFrameSync;
+  let fakeLocalStorage;
```

Comment:
Changes in this test module were necessary to get tests to pass, which feels like a code smell to me. `sidebar-content` is going to be migrated to preact soon enough that I don't know if it's worth making a fuss here or just letting it go.